bsblan/__init__.py

```diff
@@ -24,8 +24,11 @@
 #########################################################################
 from urllib.request import urlopen
 import json
+import requests
+
 from lib.model.smartplugin import *
 
+
 class Bsblan(SmartPlugin):
     ALLOW_MULTIINSTANCE = False
     PLUGIN_VERSION = '1.0.0'
```

Comment:
I think the plugin version at this point here should also updated as well as the version information in plugin.yaml